test/intl402/Locale/prototype/caseFirst/name.js

```diff
@@ -0,0 +1,25 @@
+// Copyright 2018 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-intl.locale.prototype.caseFirst
+description: >
+    Checks the "name" property of Intl.Locale.prototype.caseFirst.
+info: |
+    Unless specified otherwise in this document, the objects, functions, and constructors described in this standard are subject to the generic requirements and restrictions specified for standard built-in ECMAScript objects in the ECMAScript 2019 Language Specification, 10th edition, clause 17, or successor.
+    Every built-in function object, including constructors, that is not identified as an anonymous function has a name property whose value is a String. Unless otherwise specified, this value is the name that is given to the function in this specification. Functions that are specified as get or set accessor functions of built-in properties have "get " or "set " prepended to the property name string.
+    Unless otherwise specified, the name property of a built-in function object, if it exists, has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.
+includes: [propertyHelper.js]
+features: [Intl.Locale]
+---*/
+
+const propdesc = Object.getOwnPropertyDescriptor(Intl.Locale.prototype, "caseFirst");
+if (propdesc !== undefined) {
+  const getter = propdesc.get;
+  verifyProperty(getter, "name", {
+    value: "get caseFirst",
+    writable: false,
+    enumerable: false,
+    configurable: true,
+  });
+}
```

Comment:
What if `propdesc === undefined`?

Reply:
Then there isn't really anything to test, is there? The spec explicitly allows for it to be omitted ("This property only exists if %Locale%.[[RelevantExtensionKeys]] contains "kf".")